spec/api-crash-reporter-spec.ts

```diff
@@ -184,6 +184,15 @@ ifdescribe(!isLinuxOnArm && !process.mas && !process.env.DISABLE_CRASH_REPORTER_
       expect(crash.rendererSpecific).to.be.undefined();
     });
 
+    ifit(!isLinuxOnArm)('ensure child process args are not modified', async () => {
+      const { port, waitForCrash } = await startServer();
+      runCrashApp('node-extra-args', port);
+      const crash = await waitForCrash();
```

Comment:
a little unnecessary to actually wait for a crash here, but i suppose it doesn't matter too much.

Reply:
Updated! Using `await waitForCrash()` more intentionally now, so we don't try to detect the exitCode too early (the test waits for the child process to exit), also not saving the unneeded value